### PR TITLE
Nuncio filledassessments

### PR DESCRIPTION
**Pull request by corundumerald**

### CODE REVIEW

**Review by daniddelrio — CHANGES_REQUESTED** [review]
Good job so far! Just some comments:
1. Sorry, refer to the [updated Figma](https://www.figma.com/file/SDDsLABudWc1aIi64H8SV2/Compsat---Grade-Calculator-Website-Draft?node-id=128%3A9414) since that has the new colors of the interface which include the purple one instead of the gray.

2. Don't use global styles for this one (e.g:
``` css
table {
 ...
} 
```
This is because we're only using one CSS file, so using global styles will affect other tables made by the other devs. Better to add a div or use its parent div like
```css
.parent-class table {}
```

3. The left icon here isn't centered within the square!
Screen Shot 2020-06-12 at 3 15 16 PM

4. As much as possible, don't use properties like `left` and `top` for absolutely positioning things like the hover-icon. The hover icon is part of the Major Assessment row technically, so that div should be inside the table, in the same row as the Quizzes. Hence, include that for every Major Assessment Row, but make it so that it only appears when you hover over the Major Assessment row.

File: nuncio/style-nuncio.css [hunk 1]
```diff
@@ -18,6 +18,83 @@ html, body {
   border-radius: 10px;
 }
 
+.filled-assessments{
+  position: absolute;
+  width: 817px;
+  height: 427px;
+  left: 36px;
+  top: 258px;
```
Comment (daniddelrio) [inline nuncio/style-nuncio.css]: Remove the position: absolute, and the left and top properties here. We won't be needing them when we place everything!

**Comment by corundumerald** [comment]
Retry DeepCode

**Comment by daniddelrio** [comment]
Okay, good job! My last comment lang is to make the sections collapsible (i.e. when you click the Dropdown Arrow beside the Major Component, the Minor Assessments's visibility should be toggled). 

To accomplish that, you can make use of the [Bootstrap collapse components](https://getbootstrap.com/docs/4.5/components/collapse/). Add an ID to the Minor Assessments div, and make that the href like [here](https://getbootstrap.com/docs/4.5/components/collapse/#example).

**Comment by daniddelrio** [comment]
Last few things:

1. Make sure to add the following things so that the Bootstrap would fully work:

Add this to the end of your `<body>`:
```html
<script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
<script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js" integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI" crossorigin="anonymous"></script>
```

Add these at the start of your `<head>`
```html
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
```

2. The collapse doesn't work for me, and the reason why is because [divs don't work inside tbody](https://css-tricks.com/using-divs-inside-tables/), so the browser isn't detecting the div that has the class "collapse" and id "MinorAssessment1" surrounding the two minor assessment rows. A workaround is to add the "collapse" and "minorassessment" id to *each of the Minor Assessment rows*, then try testing to see if it works. Make it also so that when it "decollapses" (i.e. the div is hidden), the arrow changes to the arrow pointing down instead of pointing to the right.

3. Add Minor Assessments to every Major Assessment (along with the collapse feature) so that it will be easier for Santino to add the Javascript for that :)

Thanks!

**Comment by daniddelrio** [comment]
I pushed something to fix the delay in changing font styles when clicking on the arrow! It has something to do with Bootstrap's way of implementing the collapse :)) Because for a time, the "collapse" class doesn't appear for each row, making the `.assess-table .collapse td` part unused for a time. So I changed in the CSS from `collapse` to `collapsible`, and added a `collapsible` class alongside the collapse for each tr.

As for the arrow, you can check this out: https://stackoverflow.com/a/31108952
This doesn't use Javascript :)

**Comment by daniddelrio** [comment]
Hi Giana, I figured out why it's not working. When you click on the arrow, the checkbox isn't actually being checked, because of the `<a>` tag surrounding the whole label. This is my bad I didn't foresee this. Try instead the [answer with the JS here](https://stackoverflow.com/a/31108950). Lmk if it works :)

**Comment by daniddelrio** [comment]
Asking you to review this time @corundumerald :)) I edited the Javascript so that it iterates through all the elements with class names "toggle-triangle" and "the-triangle". Removed the hover as well.

Once you approve, you're free to merge!

**Pull request merged**